src/motion/basic_target_follower.cpp

```diff
@@ -3,17 +3,23 @@
 #include <ros/exception.h>
 #include <string>
 
+#include <geometry_msgs/Twist.h>
+
 #include "constructors/TwistStamped.h"
 #include "constructors/Vector3.h"
 
 BasicTargetFollower::BasicTargetFollower(float kSpeed, float kAlt, float kYaw)
     : kSpeed_(kSpeed), kAlt_(kAlt), kYaw_(kYaw) {
-  std::string velocity_topic;
-  if (!ros::param::get("velocity_topic", velocity_topic)) {
-    throw ros::Exception("Must specify angular_position_topic parameter");
-  }
+  std::string velocity_topic = "/rexrov/cmd_vel";
+  // if (!ros::param::get("velocity_topic", velocity_topic)) {
+  //   throw ros::Exception("Must specify angular_position_topic parameter");
+  // }
+  // std::string velocity_topic;
+  // if (!ros::param::get("velocity_topic", velocity_topic)) {
+  //   throw ros::Exception("Must specify angular_position_topic parameter");
+  // }
 
-  vel_pub_ = nh_.advertise<geometry_msgs::TwistStamped>(velocity_topic, 1);
+  vel_pub_ = nh_.advertise<geometry_msgs::Twist>(velocity_topic, 1);
```

Comment:
Is there a reason for this switch? If this gets switched then all the motion controllers need to be switched as well.